core/modules/simpletest/simpletest.pages.inc

```diff
@@ -347,7 +347,25 @@ function simpletest_settings_form($form, &$form_state) {
     $form['httpauth']['simpletest_httpauth_password']['#description'] = t('To change the password, enter the new password here.');
   }
 
-  return system_settings_form($form);
+  $form['actions']['#type'] = 'actions';
+  $form['actions']['submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save configuration')
+  );
+  return $form;
+}
+
+/**
+ * Form submission handler for simpletest_settings_form().
+ */
+function simpletest_settings_form_submit($form, &$form_state) {
+  config('simpletest.settings')
+    ->set('simpletest_clear_results', $form_state['values']['simpletest_clear_results'])
+    ->set('simpletest_verbose', $form_state['values']['simpletest_verbose'])
+    ->set('simpletest_method', $form_state['values']['simpletest_httpauth_method'])
+    ->set('simpletest_username', $form_state['values']['simpletest_httpauth_username'])
+    ->set('simpletest_password', $form_state['values']['simpletest_httpauth_password'])
+    ->save();
```

Comment:
Could we add `backdrop_set_message(t('The configuration options have been saved.'));` at the end of this function? Otherwise there's no message on this form.